api-docs/cloud/ref.yml

```
@@ -11143,15 +11472,41 @@ paths:
           application/json:
             schema:
               $ref: '#/components/schemas/AddResourceMemberRequestBody'
-        description: User to add as member
+        description: |
+          The user to add to the organization.
         required: true
       responses:
         '201':
           content:
             application/json:
+              examples:
+                successResponse:
+                  value:
+                    id: 09cfb87051cbe000
+                    links:
+                      self: /api/v2/users/09cfb87051cbe000
+                    name: example_user_1
+                    role: member
+                    status: active
               schema:
                 $ref: '#/components/schemas/ResourceMember'
-          description: Added to organization created
+          description: |
+            Success.
+            The response body contains the user information.
```

```suggestion
            The response body contains the user.
```
Don't have to fix this here, but I vote we should standardize on just using the resource name and eventually link to the schema object (similar to YouTube Data API).